File: ProgrammingFundamentalsExercises/DataTypesAndVariablesExercises/08.EmployeeData/EmployeeData.cs
using System;

class EmployeeData
{
    static void Main(string[] args)
    {
        string firstName = "Amanda";
        string lastName = "Jonson";
        byte age = 27;
        char gender = 'f';
        long personalId = 8306112507;
        int employeeNumber = 27563571;

        Console.WriteLine("First name: {0}", firstName);
        Console.WriteLine("Last name: {0}", lastName);
        Console.WriteLine("Age: {0}", age);
        Console.WriteLine("Gender: {0}", gender);
        Console.WriteLine("Personal ID: {0}", personalId);
        Console.WriteLine("Unique Employee number: {0}", employeeNumber);
    }
}

File: TechModule/SoftwareTechnologies/LanguageSpecifics/src/EmployeeData.java
import java.util.Scanner;

public class EmployeeData {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        String firstName = console.nextLine();
        String lastName = console.nextLine();
        int age = Integer.parseInt(console.nextLine());
        char gender = console.nextLine().charAt(0);
        long personalNumber = Long.parseLong((console.nextLine()));
        long employeeNumber = Long.parseLong((console.nextLine()));

        System.out.println("First name: " + firstName);
        System.out.println("Last name: " + lastName);
        System.out.println("Age: " + age);
        System.out.println("Gender: " + gender);
        System.out.println("Personal ID: " + personalNumber);
        System.out.println("Unique Employee number: " + employeeNumber);
    }
}